cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/Extensions.kt

```diff
@@ -294,7 +293,7 @@ private fun fancyType(
     node: HasType,
     list: MutableList<Pair<AttributedStyle, Region>>
 ) {
-    val types = TypeParser.PRIMITIVES.toMutableSet()
+    val types = outer.language?.primitiveTypes?.toMutableSet() ?: mutableSetOf()
```

Comment:
Same here

Reply:
We add a value to the set in the next line, so it has to be mutable